commercetools-models/src/test/java/io/sphere/sdk/client/JsonNodeSphereRequestIntegrationTest.java

```diff
@@ -28,4 +37,24 @@ public void execution() throws Exception {
             assertThat(jsonNode.get("name").get("en").asText()).isEqualTo(newName);
         });
     }
+
+    @Test
+    public void execution2() throws Exception {
+        withProduct(client(), product -> {
+            final String slug = product.getMasterData().getStaged().getSlug().get(Locale.ENGLISH);
+            final ProductProjectionQuery normalSphereRequest = ProductProjectionQuery.ofStaged()
+                    .withPredicates(m -> m.slug().locale(Locale.ENGLISH).is(slug))
+                    .plusExpansionPaths(m -> m.productType());
+            final JsonNodeSphereRequest jsonNodeSphereRequest = JsonNodeSphereRequest.of(normalSphereRequest);
+            assertThat(normalSphereRequest.httpRequestIntent())
+                    .as("a JsonNodeSphereRequest creates the same request to the platform, but differes in the response")
```

Comment:
It's okay if you are too lazy to modify this and wait for the CI tests to pass again, but *differs :D